### PR TITLE
Create Input Breadboard subsystem.md

### PR DESCRIPTION
**Pull request by K-Robbins-4687**

### CODE REVIEW

**Review by JesseTNRoberts — COMMENTED** [review]
I don't understand how the series connector will permit arbitrary series configurations. For instance, if I want to put 2 resistors in but not 3 or 4, how could I do that?

**Review by JesseTNRoberts — COMMENTED** [review]
What does this mean that it will be separate? It needs to be part of the car as I understand.

**Comment by K-Robbins-4687** [comment]
The team and I discussed this at length, so I'll tell you what we think. Since the car will be moving as part of the demonstration, we thought it would be best to separate the housing for the input breadboard and the car, so that the user wouldn't have to make changes to the "circuit" of a moving car.

**Comment by JesseTNRoberts** [comment]
That's not your call to make and it's not what the customer asked for. They can pick the car up or stop it to adjust the circuit.

**Review by JesseTNRoberts — COMMENTED** [review]
In person review occurred.

**Review by JesseTNRoberts — COMMENTED** [review]
This looks good! How big do the blocks need to be (precise dimensions)? How do you know the clips will hold well? What happens if the user makes a dead short (is that a problem)?  It doesn't look like you've shown that the method for measuring the resistance is sure to be able to measure the highest permitted resistance (all resistors in series).

**Review by JesseTNRoberts — COMMENTED** [review]
The Gerber files have no similarity to the proposed PCB in the top of the signoff. Further, it won't work.

File: Documentation/Signoff/Input Breadboard subsystem.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+Figure 2: Sensor Circuit
+
+![image](https://github.com/abdoulm366/TTU-Capstone--Electrical-Class-Kit/assets/158489186/1c493619-e411-4f60-878f-7f1be124eaba)
+
+Figure 3: PCB Schematic
+
+![image](https://github.com/abdoulm366/TTU-Capstone--Electrical-Class-Kit/assets/158489186/4d2e6df5-77fa-4086-aec4-3450dbacf9bf)
+
+Figure 4: PCB 3D view
+
```
Comment (micahr234) [inline Documentation/Signoff/Input Breadboard subsystem.md:33]: Where is the PCB for the detection circuit?

File: Documentation/Signoff/Input Breadboard subsystem.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+| Capacitors      | 1         | https://shorturl.at/ouxBV | $ 14.99  |
+| Diodes          | 1         | https://shorturl.at/fswBG | $ 9.99   |
+| Large Perfboard      | 1         | https://shorturl.at/xzG47 | $ 5.50  |
+| PCBs for blocks    | 20 | N/A | $30.00 |
+| Female Socket  | 120 | https://shorturl.at/cEHP4 | $13.20|
+| Jumper Wire (100 Pcs) | 1 | https://shorturl.at/noQ38 | $29.84|
+| Arduino Mega    | 1         | https://shorturl.at/htuxV | $ 48.40  |
+| Max 4167    | 1         | https://shorturl.at/EQUY7    | $ 6.18 |
+| Resistors (1% tolerance)    | 1         | https://shorturl.at/ajrDO      | $ 9.99  |
+|                 |           | Total:                    | $182.08 |
```
Comment (micahr234) [inline Documentation/Signoff/Input Breadboard subsystem.md]: Need to include manufacturer and part number in BOM.

**Review by micahr234 — APPROVED** [review]

**Pull request merged**